CONTRIBUTING.md

```diff
@@ -210,12 +215,13 @@ git pull upstream <BASE-BRANCH>
 git checkout -b <FEATURE-BRANCH>
 ```
 
-Once you've made and tested your changes, commit them with a descriptive, unique message of 74 characters or less written in the imperative tense, with a capitalized first letter and no period at the end.
+Once you've made and tested your changes, add them to the staging area with ``git add .``, and then commit them with a descriptive, unique message of 74 characters or less written in the imperative tense, with a capitalized first letter and no period at the end.
```

Comment:
```suggestion
Once you've made and tested your changes, add them to the staging area, and then commit them with a descriptive, unique message of 74 characters or less written in the imperative tense, with a capitalized first letter and no period at the end.
```

Be consistent here; the rest of this prose line describes the _what_ and _why_, while the code block below describes the _how_ of the implementation details and contains the actual code snippets